tg.go

```diff
@@ -101,6 +101,10 @@ func tgConnect(config *Config, logHandler *LogHandler) (*tgclient.TGClient, erro
 		return nil, merry.Wrap(mtproto.WrongRespError(res))
 	}
 	me := users[0].(mtproto.TL_user)
+
+	saver := &JSONFilesHistorySaver{Dirpath: config.OutDirPath}
+	saver.SaveAccount(me)
```

Comment:
No saves should occur here. `tgConnect` should just connect. However, it can return `TL_user` as the second value (before error).